source/index.html.md

```diff
@@ -50,32 +51,33 @@ mux.monitor('#my-video', {
   debug: false,
   data: {
     property_key: 'EXAMPLE_PROPERTY_KEY', // required
-    viewer_user_id: '', // ex: '12345'
     page_type: '', // (see docs) 'watchpage', 'iframe', or leave empty
+    viewer_user_id: '', // ex: '12345'
 
     // Player Metadata
     player_name: '', // ex: 'My Main Player'
     player_version: '', // ex: '1.0.0'
 
-    // Video Metadata (cleared with changeVideo)
+    // Video Metadata (cleared with 'videochange' event)
     video_id: '', // ex: 'abcd123'
     video_title: '', // ex: 'My Great Video'
     video_series: '', // ex: 'Weekly Great Videos'
     video_producer: '', // ex: 'Bob the Producer'
     video_content_type: '', // 'short', 'movie', 'episode', 'clip', 'trailer', or 'event'
-    video_language: '', // ex: 'en'
+    video_language_code: '', // ex: 'en'
     video_variant_name: '', // ex: 'Spanish Hard Subs'
     video_variant_id: '', // ex: 'abcd1234'
-    video_duration: '', // in seconds, ex: 120.0
+    video_duration: '', // in milliseconds, ex: 120000
     video_stream_type: '', // 'live' or 'on-demand'
```

Comment:
We should probably remove `video_stream_type` until we know exactly how we want to use it.